src/pages/index.js

```diff
@@ -38,21 +80,48 @@ const createCard = (cardData) => {
 const addCardObject = {
   popupSelector: "#add-card-modal",
   handleFormSubmit: (inputValues) => {
-    const cardElement = createCard(inputValues);
-    addCardForm.popupForm.reset();
-    addCardFormValidator.toggleButtonState();
```

Comment:
Please, delete `this.toggleButtonState();` from line 80 in `FormValidator.js` because now you can't toggle the button immediately after submitting the form: you need to wait till the server successfully responds in `then` and only then disable the button otherwise you can't send the form for the 2nd time in case of a server error

Reply:
Now the program will only effect the form after a correct response from the server.